client-data/tools/pencil/pencil.js

```diff
@@ -38,6 +43,29 @@
 		this.y = y;
 	}
 
+	function hideMarker() {
+		// hide cursor again unless it is set to always be shown
+		Tools.showMarker = Tools.alwaysShowMarker;
+		if (!Tools.showMarker) {
+			var cursor = Tools.svg.getElementById("mycursor");
+			if (cursor) {
+				cursor.remove();
+			}
+		}
+	}
+
+	function onStart(){
+		if(curPen === "whiteout"){
+			Tools.showMarker = true;
+		}
+	}
```

Comment:
I don't think the cursor should be accessed from other tools. You can use the `showMarker` attribute of individual tools that you introduced in the cursor PR. You can use something like `pencilTool.showMarker = False`

Reply:
I have decided that since we were displaying the cursor anyway (`alwaysShowMarker` was set to `true`) we don't need any of the code that was checking if it should hide the cursor